src/utils/dates.js

```diff
@@ -10,3 +10,23 @@ export const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
```

Comment:
Appreciate the way both of these functions are written - easy to follow logic and variable names. Nice work!